Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
 	OutputLine(NULL, "…『鬼隠し』でも、今回の『綿流し』でも。",
 		   NULL, " ...Both last time in Onikakushi, and this time in Watanagashi.", Line_WaitForInput);
-	PlaySE(4, "S02/00/440900118_2", 128, 64);
+	PlaySE(4, "S02/00/440900119", 128, 64);
```

Comment:
That's odd, I'm not finding 440900118_2 and 440900119 anywhere. I even searched through the first game SE folder, but it's not there. Do you have these files? Maybe my folder is incomplete.